core/templates/dev/head/pages/admin/admin_navbar_directive.html

```diff
@@ -1,11 +1,15 @@
 <style>
+  .nav > li > a:focus {
```

Comment:
This is too broad, prefix it with `admin-navbar` like in the other CSS selectors below.

Reply:
This is actually a hack to avoid applying of this selector from generated_styles.lib and oppia.css.

Reply:
I think you need a comment above this line explaining that, then; it's not clear to the reader.

That said, I don't understand why prefixing it with `admin-navbar` doesn't work. Wouldn't that make the CSS rule even more specific, and thus higher priority?

Reply:
Will not work beacause it will get higher priority than 
```admin-navbar .oppia-navbar-tabs-admin > li > .active```
 thus will override background property when the tab is both focused and active. But perhaps we can do it this way 
```
<html>admin-navbar .nav > li > a:focus:not(.active)</html>
```

Reply:
Ah, I see, thank you for explaining. Yep, I think what you propose is fine!

Reply:
Just noticed that ``` admin-navbar .nav > li > a:focus:not(.active) ``` is not working ideal, the tab get rehighlighted to blue and than back to white on mouse click. Commited another fix.